flink-core/src/main/java/org/apache/flink/api/common/io/SinkUtils.java

```diff
@@ -25,6 +27,7 @@
 import java.util.concurrent.TimeoutException;
 
 /** Utility class for sinks. */
+@Internal
```

Comment:
Utils used by connectors shouldn't be `@Internal`, because it conflicts with the plans to externalize connectors.

Reply:
thanks for the precision. Noted